src/ipfsd-daemon.js

```diff
@@ -19,6 +19,7 @@ const daemonLog = {
   info: logger('ipfsd-ctl:daemon:stdout'),
   err: logger('ipfsd-ctl:daemon:stderr')
 }
+const rpcModuleLogger = logger('ipfsd-ctl:daemon:rpcModule:stdout')
```

Comment:
The `stdout`/`stderr` suffix is because we're logging the output of an external process, that's not the case for this logger so it should be shortened:

```suggestion
const rpcModuleLogger = logger('ipfsd-ctl:daemon')
```